src/main/java/com/joss/conductor/mobile/annotation/IOSId.java

```diff
@@ -0,0 +1,17 @@
+package com.joss.conductor.mobile.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Created on 3/14/18.
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface IOSId {
+
+    String value() default "";
```

Comment:
Maybe change to `name` since that should be the default for iOS?

Reply:
see above comments on shorthand